backend/scripts/bootstrap.js

```diff
@@ -89,18 +110,24 @@ const getCryptosHours = async () => {
     await client
         .query(query)
         .catch(e => console.error(e.stack))
+    cryptoHoursBar.stop()
 };
 
 
 // GET EACH CRYPTO VALUES FOR 120 LAST MINUTS
 const getCryptosMinuts = async () => {
     let query = 'INSERT INTO crypto_history (crypto_id, period, timestamp, open, high, low, close) VALUES \n';
+
+    const cryptoMinutsBar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
+    cryptoMinutsBar.start(numberOfMinuts * cryptos.length, 0);
+
     for (const [index, crypto] of cryptos.entries()) {
-        const url = `https://min-api.cryptocompare.com/data/v2/histominute?fsym=${crypto}&tsym=${symbol}&limit=120&api_key=${api_key}`;
+        const url = `https://min-api.cryptocompare.com/data/v2/histominute?fsym=${crypto}&tsym=${symbol}&limit=${numberOfMinuts}&api_key=${api_key}`;
         await axios.get(url)
             .then(response => {
                 const minuts = response.data.Data.Data;
```

Comment:
```suggestion
                const minuts = response.data.Data.Data;
```
nit : should be names `minutes`